lib/apm.js

```diff
@@ -342,7 +342,7 @@ var Instrumentation = function(core, options, callback) {
           }
 
           if(cmd.maxTimeMS) command.maxTimeMS = cmd.maxTimeMS;
-        } else if(x == '_killcursors') {
+        } else if(x == '_killcursor') {
```

Comment:
This does not make a difference for fixing this particular issue, but the methods array above has '_killcursor' (not plural).